pandas/core/internals/managers.py

```diff
@@ -554,13 +554,6 @@ def quantile(
             for blk in self.blocks
         ]
 
-        if transposed:
```

Comment:
The `transposed` keyword is no longer used after removing this? (so the keyword can be removed as well)